src/sync_drive.py

```diff
@@ -27,8 +32,11 @@ def wanted_file(filters, file_path):
     return False
 
 
-def wanted_folder(filters, root, folder_path):
+def wanted_folder(filters, ignore, root, folder_path):
     """Check if folder is wanted."""
+    if ignore:
+        if PathSpec.from_lines("gitwildmatch", ignore).match_file(f"{folder_path}/foo.bar"):
```

Comment:
`f"{folder_path}/foo.bar"` shouldn't this be `folder_path`?

Reply:
This is necessary 😕. If you remove `/foo.bar` the tests will not pass 
If you have another suggestion, I can try it.

Reply:
Looks like https://github.com/cpburnz/python-pathspec/issues/65 is causing the tests to fail. Seems like `pathspec` isn't the right fit for our needs. 😕

It might be easier to use something like https://pymotw.com/3/glob/ or our own implementation to check if entries in `ignore` list matches the path and then `return False` to ignore it.

Reply:
Glob is useful to find filenames, but do not seems helpful for testing path

Reply:
Looks like `glob` internally uses `fnmatch`. `fnmatch` seems to be a good fit for this feature.